image/postgres/rhtap.Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as builder
```

Comment:
Based on https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/blob/rhacs-1.0-rhel-8/distgit/containers/rhacs-central-db/Dockerfile.in

Reply:
Up to you, of course, but you can avoid copying the entire source code and having this `builder` stage altogether by adjusting `COPY` commands in the `final` stage: take files from the context instead of the `builder` stage.

The `builder` exists in downstream dockerfile because it's most canonical to use provided environment variables (`$REMOTE_SOURCES` and `$REMOTE_SOURCES_DIR`) that way. Since these variables don't exist and aren't needed in RHTAP, we can do it simpler here.

Reply:
Simplified to one stage.